scrapers_next/hi/committees.py

```diff
@@ -0,0 +1,119 @@
+from spatula import HtmlPage, XPath, URL, SelectorError
+from openstates.models import ScrapeCommittee
+
+
+# All committiees are listed on the same page
+class CommitteeList(HtmlPage):
+    source = "https://www.capitol.hawaii.gov/comminfolist.aspx"
+
+    def process_page(self):
+
+        # List of all committees on the page
+        committees = XPath(
+            "//div[@id='tabContent']//div[contains(@class,'justify-content-begin')]"
+        ).match(self.root)
+
+        for com in committees:
+            yield self.process_committee(com)
+
+    def process_committee(self, com_element):
+        href = XPath("./h3/a/@href").match_one(com_element)
+        name = XPath("./span/text()").match_one(com_element).strip()
+        return CommitteeDetails(
+            {"name": name, "list_url": self.source.url}, source=URL(href, timeout=30)
+        )
+        print(name, href)
```

Comment:
`print` after return will never run. Let's either comment or remove it.

Reply:
Thanks, I forgot to remove that. Fixed with https://github.com/openstates/openstates-scrapers/pull/4477/commits/1e168928cf300fa567e79dfd68b346ac227ff7d1